test.scf

```diff
@@ -125,3 +125,10 @@ try profiling
     stats
     $ rm stats file.svg
     $ ls
+
+be robust with regard to branch names that contain special characters
```

Comment:
This look convenient, but seems to be somewhat misusing the scruf testing framework since it's not strictly testing features of the command line interface. But, this is probably a lot harder/laborious to do in "regular" python test code with `unitest` and friends?

Reply:
It's mostly a crash test for now, true, yes. We can make a deal about re-writing it outside of scruf after 1.0.0 if you like.

Reply:
ok, let's include it since it solves the problem in a pragmatic way!

Reply:
Thank you!